src/routes/guides/styling-components/data.json

```
@@ -6,6 +6,7 @@
 		"css-modules.mdx",
 		"macaron.mdx",
 		"tailwind.mdx",
-		"uno.mdx"
+		"uno.mdx",
+		"tailwind-v3.mdx"
```

```suggestion
		"uno.mdx"
```

I suggest removing the v3 docs from here. There is a link to the v3 docs in the v4 docs for the people who look for it.